builders/build/eth-api/dev-env/brownie.md

```diff
@@ -274,7 +264,7 @@ You can now deploy the `Box.sol` contract using the `run` command and specifying
 === "Moonbeam Dev Node"
 
     ```bash
-    brownie run scripts/deploy.py --network moonbeam-dev
+    brownie run scripts/deploy.py --network moonbase
```

Comment:
we have to fix this if this is the case, otherwise it's just going to cause confusion.

but oddly enough, it doesn't seam to be the case for me. i even uninstall brownie and reinstalled it to make sure but this is what i see


<img width="201" alt="Screenshot 2024-01-09 at 11 08 49 PM" src="https://github.com/moonbeam-foundation/moonbeam-docs/assets/26533957/91e8a1de-0c2a-46f5-bde5-ac634e04437f">

Reply:
https://github.com/eth-brownie/brownie/blob/bc7b511583060fdaff1d4b5269aedcc1cb710bc6/brownie/data/network-config.yaml#L134-L155

Reply:
actually looks like brownie still doesn't come with a dev node as an available network out of the box

Reply:
yeah, so I realized even though I had uninstalled and reinstalled brownie, there were some things cached - including my network settings and accounts.

so I started over from scratch on my vps. installing brownie was a chore, because of some dependency conflicts. I ran `pipx install eth-brownie` and got the following error:

```
Fatal error from pip prevented installation. Full pip output in file:
    /root/.local/state/pipx/log/cmd_2024-01-09_23.18.37_pip_errors.log

pip seemed to fail to build package:
    pyyaml==5.4.1

Some possibly relevant errors from pip install:
    error: subprocess-exited-with-error
    AttributeError: cython_sources

Error installing eth-brownie.
```

So then I tried installing brownie with pip instead of pipx and got a bit more of a detailed error.

After some googling, I found a solution and to successfully install brownie i had to use the following command:

```
pip3 install wheel && pip3 install --no-build-isolation "Cython<3" "pyyaml==5.4.1" && pip3 install --upgrade --no-cache-dir eth-brownie
```

This whole debacle led me into realizing that isn't really actively being maintained. So maybe we should consider sunsetting this page in the near future. I'll have to take a look to see how many visits this page gets. But lots of signs pointing to using ApeWorX instead: https://github.com/ApeWorX/ape

ANYWAYS ahah sorry on a rant here. When I did finally run `brownie list networks` these are the networks, which aligns with the `network-config.yaml` file shared in my previous comment:

<img width="271" alt="Screenshot 2024-01-09 at 11 37 27 PM" src="https://github.com/moonbeam-foundation/moonbeam-docs/assets/26533957/77cd1123-778d-48cc-9dc8-8123d80f6432">

so with all this being said, brownie does not support a moonbeam dev node out of the box